HousingSearchApi/V2/Gateways/SearchGateway.cs

```diff
@@ -49,17 +49,32 @@ public async Task<SearchResponseDto> Search(string indexName, SearchParametersDt
                 SearchOperations.MatchPhrasePrefix(searchParams.SearchText, fieldName: addressFieldName, boost: 10),
                 SearchOperations.WildcardMatch(searchParams.SearchText, fieldName: addressFieldName, boost: 5),
             });
+            shouldOperations.AddRange(defaultShouldOperations);
         }
         else if (indexName == "persons")
         {
-            var nameFields = new List<string> { "firstname", "surname" };
+            Fields nameFields = new[] { "title", "firstname", "surname" };
+            Fields addressFields = new[] {
+                "tenures.id",
+                "tenures.assetFullAddress",
+                "tenures.type",
+                "tenures.uprn",
+                "tenures.propertyReference",
+                "tenures.assetId",
+                "tenures.paymentReference",
+            };
+
             shouldOperations.AddRange(new[]
             {
-                SearchOperations.SearchWithWildcardQuery(searchParams.SearchText, boost: 10, fields: nameFields),
-                SearchOperations.SearchWithExactQuery(searchParams.SearchText, boost: 6, fields: nameFields),
+                SearchOperations.MultiMatchSingleField(searchParams.SearchText, boost: 6),
+                SearchOperations.MultiMatchMostFields(searchParams.SearchText, boost: 10, fields: nameFields),
+                SearchOperations.NestedMultiMatch(searchParams.SearchText, "tenures", addressFields, boost: 10),
```

Comment:
The key functional change in this PR - these operations are designed to work well with the new index mapping.
- MultiMatchSingleField: Allows matching an arbitrary field exactly
- MultiMatchMostFields with nameFields: Allows matching first and last names with arbitrary parts (in practice helpful for missing out parts of names like middle names)
- NestedMultiMatch with addressFields: Allows search on and across the tenure address fields